### DISCUSSION TITLE
"Go Offline" from collab channels without leaving or signing out

### DISCUSSION THREAD

**Discussion by jansol** [zed-industries/zed#23947] [Feature Requests]
Analogous to "offline" status in Steam.

I was looking at Zed constantly using 0.10% CPU in the background and remembered that I am still connected to the Zed channel.

It would be nice to have an option to temporarily disconnect from all collab without immediately invalidating my login token. Stuff like installing extensions should still be available, though, as those are manually triggered actions.

**Comment by CurbaiCode** [comment]
#10170 looks like a similar idea.

**Comment by maxdeviant** [comment]
This sounds like an interesting idea. Going to move to a discussion.
